README.md

```diff
@@ -21,10 +21,10 @@ etc.), regardless of what version of Node.js is actually installed on your syste
 
 ## Installation
 
-You can install `node-gyp` using `npm`:
+You can install `node-gyp` using `npm`(run as root/admin):
 
 ``` bash
-$ npm install -g node-gyp
```

Comment:
also this change should be reverted or you're going to need to do it for all of the bash blocks in the README (which I'm not opposed to)